src/YQReplacePoint.cc

```diff
@@ -42,10 +42,21 @@ void YQReplacePoint::showChild()
 {
     YWidget * child = firstChild();
 
+    /* This needs to be done so the UI won't crash if the are no children. */
     if ( child )
     {
-	QWidget * qChild = (QWidget *) child->widgetRep();
-	qChild->show();
+        QWidget * qChild = (QWidget *) child->widgetRep();
+        qChild->show();
+
+        /* Refresh the whole UI so it doesn't get stalled in some situations. */
```

Comment:
and if such issue do not exists, is there some email discussion about this or something similar?